docassemble_base/docassemble/base/filter.py

```diff
@@ -980,7 +981,17 @@ def image_url(file_reference, alt_text, width, emoji=False, question=None, exter
         else:
             extra_class = ''
         if file_info.get('extension', '') in ('png', 'jpg', 'gif', 'svg', 'jpe', 'jpeg'):
-            return '<img ' + alt_text + 'class="daicon daimageref' + extra_class + '" style="' + width_string + '" src="' + the_url + '"/>'
+            try:
+                if file_info.get('extension', '') == 'svg':
+                    attributes = ET.parse(file_info['fullpath']).getroot().attrib
+                    layout_width = attributes['width']
+                    layout_height = attributes['height']
+                else:
+                    im = PIL.Image.open(file_info['fullpath'])
+                    layout_width, layout_height = im.size
```

Comment:
```suggestion
                    with PIL.Image.open(file_info['fullpath']) as im:
                        layout_width, layout_height = im.size
```
Will automatically close the image.